oci8_sql_test.go

```diff
@@ -468,7 +468,7 @@ end;
 	_, err = stmt.ExecContext(ctx)
 	cancel()
 	expected := "ORA-01013"
-	if err == nil || len(err.Error()) < len(expected) || err.Error()[:len(expected)] != expected {
+	if err == nil || len(err.Error()) < len(expected) || !strings.Contains(err.Error(), expected) {
```

Comment:
Is there a reason this changed? Is ORA-01013 no longer in the front?

Reply:
Yes, there were cases when it was a chain of errors, and it wasn't in front. I am trying to reproduce it now, and I'll show you the error.

Reply:
Not a huge deal, would be curious.

Reply:
was able to reproduce it 
```--- FAIL: TestContextTimeoutBreak (0.06s)
    oci8_sql_test.go:472: stmt exec - expected: ORA-01013 - received: ORA-00604: error occurred at recursive SQL level 1
        ORA-01013: user requested cancel of current operation
```